src/GraphQL/Mapper/ImageAssetMapperStrategyInterface.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace Ibexa\GraphQL\GraphQL\Mapper;
+
+use eZ\Publish\API\Repository\Values\Content\Field;
+use eZ\Publish\Core\FieldType\ImageAsset;
+
+interface ImageAssetMapperStrategyInterface
+{
+    public function canProcess(ImageAsset\Value $value): bool;
```

Comment:
Same as above but for `Value`.

Reply:
I'm sticking to one convention, it's even in the same file: https://github.com/ezsystems/ezplatform-graphql/blob/2.3/src/GraphQL/Resolver/ImageAssetFieldResolver.php#L48

Reply:
I agree with @barw4. There are multiple places in codebase where we are doing it. It helps improve readability of the code.